app.py

```diff
@@ -1,250 +1,285 @@
 import os
 import pickle
 import streamlit as st
+import pandas as pd
+import plotly.express as px
+import numpy as np
+from datetime import datetime, timedelta, date
 from streamlit_option_menu import option_menu
 
-# Set page configuration
-st.set_page_config(page_title="Health Assistant",
-                   layout="wide",
-                   page_icon="🧑‍⚕️")
-
-    
-# getting the working directory of the main.py
-working_dir = os.path.dirname(os.path.abspath(__file__))
-
-# loading the saved models
-
-diabetes_model = pickle.load(open(f'{working_dir}/Models/diab_model.sav', 'rb'))
-
-heart_disease_model = pickle.load(open(f'{working_dir}/Models/heart_disease_model.sav', 'rb'))
-
-parkinsons_model = pickle.load(open(f'{working_dir}/Models/parkinsons_model.sav', 'rb'))
-
-# sidebar for navigation
+# ─── 1) Must be first ─────────────────────────────────────────────────────────
+st.set_page_config(
+    page_title="Health Assistant",
+    layout="wide",
+    page_icon="🧑‍⚕️"
+)
+
+# ─── 2) Custom CSS ─────────────────────────────────────────────────────────────
+st.markdown("""
+<style>
+  .stApp { background-color: #F5F5F5; }
```

Comment:
This will break the default theme for the dark background asthe streamlit will force the dark theme with gray text but your theme will make grey forcefully . thus the text will not be visible